ebsco/edsapi-koha-plugin
# Search load fix

This PR addresses issue #94 with the search taking a long time to load.

Many thanks to @kidclamp and @AndrewIsh for the help identifying this issue and potential solutions!

To Test:

Install on your Koha 20.05 server.
BE SURE that you go into "Configure" and click "save" after install, even if it stores old values.  This builds the templates needed.
Check that everything is functioning as expected and that the dropdown is populating as needed.
